customer-service/src/main/scala/infrastructure/repositories/CustomerDetailsHttpRepository.scala

```diff
@@ -78,7 +74,7 @@ class CustomerDetailsHttpRepository[F[_]: Async](config: CustomerDetailsServiceC
       .map(_.transformInto[Option[CustomerDetails]])
 
     val customerDetailsLegacyRetry =
-      retryingOnAllErrors(customerDetailsLegacy, legacyRepositoryRetryPolicy).handleError(_ => None)
+      RetryHelpers.retryOnAllErrors(customerDetailsLegacy, Duration.apply(2, "second"), 4).handleError(_ => None)
```

Comment:
`import scala.concurrent.duration.DurationInt` and enjoy your life with `2.seconds` instead of `Duration.apply(2, "second")`